### PR TITLE
feat:faq-section

### PR DESCRIPTION
**Pull request by ajeeshRS**
### Added Faq-section

Screenshot 2024-09-12 at 9 54 57 PM
Screenshot 2024-09-12 at 10 01 54 PM

 Let me know if it requires any further improvements.

### CODE REVIEW

File: src/lib/constant/faqs.constants.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+export const faqData = [
```
Comment (VineeTagarwaL-code) [inline src/lib/constant/faqs.constants.ts:1]: should add types here tbh

**Review by VineeTagarwaL-code — CHANGES_REQUESTED** [review]

File: src/lib/constant/faqs.constants.ts (hunk 1 above)
Comment (ajeeshRS) [reply src/lib/constant/faqs.constants.ts]: done

**Comment by VineeTagarwaL-code** [comment]
rather than cards i will strongly suggest two things 

1. make it accordian 
2. have this in the main landing page only just after the listed jobs 
3. when clicked on the navbar make it autoscroll to the faq component in the landing page itself 

that's all 

after you make these changes, post a video of it ... if you have any doubts let me know

**Comment by ajeeshRS** [comment]
Hey @VineeTagarwaL-code 
Here is the video of updated faq-section
https://github.com/user-attachments/assets/a8e4cfb7-d591-4ff8-9de0-cca83f146f1b

**Comment by VineeTagarwaL-code** [comment]
Looks so cool man !!!, please also give the mobile ui

**Pull request merged**